integration/shared/isolated/network_policies_command_test.go

```diff
@@ -72,8 +72,47 @@ var _ = Describe("network-policies command", func() {
 				username, _ := helpers.GetCredentials()
 				Eventually(session).Should(Say(`Listing network policies in org %s / space %s as %s\.\.\.`, orgName, spaceName, username))
 				Consistently(session).ShouldNot(Say("OK"))
-				Eventually(session).Should(Say(`source\s+destination\s+protocol\s+ports`))
-				Eventually(session).Should(Say(`%s\s+%s\s+tcp\s+8080[^-]`, appName, appName))
+				Eventually(session).Should(Say(`source\s+destination\s+protocol\s+ports\s+destination space\s+destination org`))
+				Eventually(session).Should(Say(`%s\s+%s\s+tcp\s+8080\s+%s\s+%s`, appName, appName, spaceName, orgName))
+				Eventually(session).Should(Exit(0))
+			})
+		})
+
+		When("policy has a destination in another org and space", func() {
+			var (
+				destOrg   string
+				destSpace string
+				destApp   string
+			)
+
+			BeforeEach(func() {
+				destOrg = helpers.NewOrgName()
+				destSpace = helpers.NewSpaceName()
+				destApp = helpers.PrefixedRandomName("destapp")
+
+				helpers.SetupCF(destOrg, destSpace)
+
+				helpers.WithHelloWorldApp(func(appDir string) {
+					Eventually(helpers.CF("push", destApp, "-p", appDir, "-b", "staticfile_buildpack", "--no-start")).Should(Exit(0))
+				})
+
+				helpers.SetupCF(orgName, spaceName)
+
+				session := helpers.CF("add-network-policy", appName, "--destination-app", destApp, "-o", destOrg, "-s", destSpace)
+				username, _ := helpers.GetCredentials()
+				Eventually(session).Should(Say(`Adding network policy from app %s in org %s / space %s to app %s in org %s / space %s as %s\.\.\.`, appName, orgName, spaceName, destApp, destOrg, destSpace, username))
+				Eventually(session).Should(Say("OK"))
+				Eventually(session).Should(Exit(0))
+			})
+
+			It("lists the policy", func() {
+				session := helpers.CF("network-policies")
+
+				username, _ := helpers.GetCredentials()
+				Eventually(session).Should(Say(`Listing network policies in org %s / space %s as %s\.\.\.`, orgName, spaceName, username))
+				Consistently(session).ShouldNot(Say("OK"))
```

Comment:
See comment in `add_network_policy_command_test.go`.

Reply:
Agree, will remove.